talk/expert/variadictemplate.tex

```diff
@@ -121,7 +121,7 @@
 
 \begin{frame}[fragile]
   \frametitlecpp[11]{Variadic class template}
-  \begin{block}{The tuple example, simplified}
+  \begin{block}{The tuple example, simplified \godboltLink{https://godbolt.org/z/nxjja4bob}}
```

Comment:
I don't see how to go to cppinsights from godbolt. I had this is #366:
https://cppinsights.io/s/600802a9

Reply:
Just click on the magnifier icon :-)